scripts/aws/cloudformation/tasking-manager.template.js

```diff
@@ -359,80 +362,155 @@ const Resources = {
+                Value: 'dakota.benjamin and yogeshgirikumar'
+              },
+              {
+                Key: 'environment',
+                Value: cf.ref('NetworkEnvironment')
+              },
+            ]
+          }
+        ],
+        UserData: cf.userData([
+          '#!/bin/bash',
+          'set -x',
+          'sleep 10',
+          'export DEBIAN_FRONTEND=noninteractive',
+          'export LC_ALL="en_US.UTF-8"',
+          'export LC_CTYPE="en_US.UTF-8"',
+          'dpkg-reconfigure --frontend=noninteractive locales',
+          'sudo apt-get -q -y update',
+          'sudo DEBIAN_FRONTEND=noninteractive apt-get -q -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" dist-upgrade',
+          'sudo apt-get -q -y install awscli curl git gnupg ruby wget',
+          'sudo apt-get -q -y install python3.9 python3-pip python3-psycopg2 python-is-python3',
+          'sudo apt-get -q -y install postgresql-client postgis libpq-dev',
+          'sudo apt-get -q -y install libxml2 libxml2-dev',
+          'sudo apt-get -q -y install libgeos-3.9.0 libgeos-dev',
+          'sudo apt-get -q -y install libproj19 libproj-dev',
+          'sudo apt-get -q -y install libjson-c-dev',
+          'git clone --recursive https://github.com/hotosm/tasking-manager.git /opt/tasking-manager',
+          'cd /opt/tasking-manager/',
+          cf.sub('git reset --hard ${GitSha}'),
+          'pip install --upgrade pip pdm==2.7.4',
+          'wget -6 https://s3.dualstack.us-east-1.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-py3-latest.tar.gz -O /tmp/aws-cfn-bootstrap-py3-latest.tar.gz',
+          'pip install /tmp/aws-cfn-bootstrap-py3-latest.tar.gz',
+          'pdm install',
+          'eval "$(pdm venv activate)"',
+          'echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf',
+          'export LC_ALL=C',
+          'wget -6 https://s3.dualstack.us-east-1.amazonaws.com/amazoncloudwatch-agent/ubuntu/amd64/latest/amazon-cloudwatch-agent.deb -O /tmp/amazon-cloudwatch-agent.deb',
+          'dpkg -i /tmp/amazon-cloudwatch-agent.deb',
+          'echo "Exporting environment variables:"',
+          cf.sub('export NEW_RELIC_LICENSE=${NewRelicLicense}'),
+          cf.join('', ['export POSTGRES_ENDPOINT=', cf.getAtt('TaskingManagerRDS','Endpoint.Address')]),
+          cf.sub('export POSTGRES_DB=${PostgresDB}'),
+          cf.sub('export POSTGRES_PASSWORD="${PostgresPassword}"'),
+          cf.sub('export POSTGRES_USER="${PostgresUser}"'),
+          cf.sub('export TM_APP_BASE_URL="${TaskingManagerAppBaseUrl}"'),
+          cf.sub('export TM_ENVIRONMENT="${AWS::StackName}"'),
+          cf.sub('export TM_CLIENT_ID="${TaskingManagerOAuthClientID}"'),
+          cf.sub('export TM_CLIENT_SECRET="${TaskingManagerOAuthClientSecret}"'),
+          cf.sub('export TM_REDIRECT_URI="${TaskingManagerAppBaseUrl}/authorized"'),
+          'export TM_SCOPE="read_prefs write_api"',
```

Comment:
Can be removed since the same value is set as default in the backend config file.